stable/fairwinds-insights/templates/_env.yaml

```diff
@@ -0,0 +1,125 @@
+{{ define "env" }}
```

Comment:
I totally get why you broke this out, that's a lot of Env here. 

I'm curious though, do all deployments (now and future) need all these environment variables? I would be worried about accidentally leaking things like password to workloads that don't need it.

Reply:
That's a good question. `now` the answer is yes, for the most part. I could see it getting misused in the future though.

Maybe we should have a few different envs that can get included down the line